kobo/settings/base.py

```diff
@@ -274,36 +274,48 @@
     ),
     'USER_METADATA_FIELDS': (
         json.dumps([
-            {'name': 'organization', 'required': False},
-            {'name': 'organization_website', 'required': False},
-            {'name': 'sector', 'required': False},
-            {'name': 'gender', 'required': False},
-            {'name': 'bio', 'required': False},
-            {'name': 'city', 'required': False},
-            {'name': 'country', 'required': False},
-            {'name': 'twitter', 'required': False},
-            {'name': 'linkedin', 'required': False},
-            {'name': 'instagram', 'required': False},
+            {'name': 'full_name', 'required': False,},
+            {'name': 'organization', 'required': False,},
+            {'name': 'organization_website', 'required': False,},
+            {'name': 'sector', 'required': False,},
+            {'name': 'gender', 'required': False,},
+            {'name': 'bio', 'required': False,},
+            {'name': 'city', 'required': False,},
+            {'name': 'country', 'required': False,},
+            {'name': 'twitter', 'required': False,},
+            {'name': 'linkedin', 'required': False,},
+            {'name': 'instagram', 'required': False,},
         ]),
         # The available fields are hard-coded in the front end
         'Display (and optionally require) these metadata fields for users. '
-        "Possible fields are 'organization', 'organization_website', "
+        "Possible fields are 'full_name', 'organization', 'organization_website', "
         "'sector', 'gender', 'bio', 'city', 'country', 'twitter', 'linkedin', "
-        "and 'instagram'",
+        "and 'instagram'.\n\r"
```

Comment:
I've never seen `\n\r` before. The reverse of that, `\r\n` is common to indicate a single line break (new line + carriage return) and is part of many [standards](https://github.com/kobotoolbox/kpi/pull/3825#issuecomment-1155501572), but at some point django-constance [changed](https://github.com/kobotoolbox/kpi/pull/3831) to use only `\n` for line breaks in **values**. However, that doesn't necessarily mean anything for these field *descriptions*.

I was curious so I did some tests: it looks like your `\n\r` is being interpreted as 2 consecutive newlines and translated to `<br><br>` in the Django Admin HTML. I tested and the same thing can be achieved with `\n\n`, which should be easier for a human reader to understand as two consecutive newlines.